leapp/cli/upgrade/__init__.py

```diff
@@ -117,6 +117,21 @@ def generate_report_files(context):
     generate_report_file(messages, context, report_txt)
 
 
+def warn_if_unsupported():
+    env = os.environ
+    devel_vars = {k: env[k] for k in env if k.startswith('LEAPP_DEVEL_')}
```

Comment:
Hmmm iirc the final agreement was to have some special variable that would control if the upgrade is supported, not to check LEAPP_DEVEL_* vars. @vinzenz @shaded-enmity ?

Reply:
Checking for those is ok, but it should not inhibit when the "LEAPP_I_UNDERSTAND_THE_SYSTEM_IS_UNSUPPORTED" (or something like this) is set to 1

Reply:
It's like that now.